docs/building.md

```diff
@@ -23,25 +39,58 @@ cargo test
 
 ## Android development
 
-Roughly you need Java 8, the Android SDK, the NDK R20 and a bunch of environment variables.  
-Your best friend is the following command:
+### Java 8
+
+Using any other version than 8 __will__ mess up your build and is not supported.  
+Please ensure `JAVA_HOME` points to the JDK 8 installation directory.
+
+### Setting up the Android SDK
+
+#### Through Android Studio
+
+1. Download and install Android Studio and let it install the SDK for you.
+1. In the "SDK Manager" window, find out the location of the SDK.
+1. Set `ANDROID_SDK_ROOT` and `ANDROID_HOME` (technically deprecated 🤷‍♂️) to this location and add it to your rc file.
+
```

Comment:
Note that we do not have to install the NDK manually anymore thanks to #3222, `verify-android-environment.sh` will take care of it.